plugins/org.eclipse.sirius.diagram.sequence.ui/src/org/eclipse/sirius/diagram/sequence/ui/tool/internal/layout/SequenceZOrderingRefresher.java

```diff
@@ -79,6 +86,35 @@ private void moveObservationPointToFront() {
      */
     private void moveCombinedFragmentsToBack() {
         moveParts(CombinedFragmentEditPart.class, false, new CombinedFragmentVerticalPositionFunction());
+
+    }
+
+    /**
+     * Shift the container, representing the interaction, if it exists, before every other children so they appear
+     * visually behind the rest.
+     */
+    private void moveInteractionContainerToBack() {
+        Optional<DNodeContainerEditPart> optionalInteractionContainer = getInteractionContainer(sequenceDiagramPart.getChildren());
+        if (optionalInteractionContainer.isPresent()) {
+            sequenceDiagramPart.reorderChild(optionalInteractionContainer.get(), 0);
+        }
+    }
+
+    /**
+     * Return the container representing the interaction if it exists.<BR>
+     * TODO: To adapt according to a specific mapping when it will be created.
```

Comment:
Is the TODO still relevant?

Reply:
You are right, I forgot to remove the todo once I introduced the custom edit part.